### DISCUSSION TITLE
Add a new function

### DISCUSSION THREAD

**Discussion by mimi-ac** [MiczFlor/RPi-Jukebox-RFID#1938] [Q&A]
Hello everybody,

i tried to add an new function to rfid_trigger_play.conf.

But the new function isn´t shown in the WebUI.

For example I add:

**rfid_trigger_play.conf**

```
## Own Commands
### My New Command
CMDNEW="34324334" 
```

**rfid_trigger_play.sh:**
```

        $CMDNEW)
                echo 'Test'
                ;;


```

What went wrong?

Thankx

**Comment by MiczFlor** [comment]
99.9% sure that you need to make the change in the file `rfid_trigger_play.conf.sample`. See this line:
https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/5be26c528f7731cd4fd1d4ba23913e5f2622d8fa/htdocs/cardRegisterNew.php#L140
